new/components/vcloud/update-vcloud-vm.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash -e
+
+uname -a
+
+# update system
+yum update --skip-broken -y
```

Comment:
why are these required?
We want to provide an 'offline' experience meaning that yum update might not be accessible. Also, installing compilers is in general something that we're trying to avoid, thanks to which we're providing all compilation-requiring python modules already compiled as wheels for Centos 7. Is this because the vcloud plugin itself required compilation? If so, @idanmo , please advise.